src/components/LogIn/LogIn.css

```diff
@@ -1,9 +1,59 @@
-body{
+.loginBody{
     background-color: black;
     overflow: hidden;
     margin: 0;
 }
 
```

Comment:
Are we going to want some of the CSS properties to be global to the entire application? Since the button and other components were moved to the util folder to be utilized by the entire application should we add these CSS properties for them to a global CSS file?